test/integration/components/droplist.js

```diff
@@ -165,4 +165,58 @@ context('Droplist', function() {
       .get('.picklist')
       .should('not.exist');
   });
+
+  specify('isCheckable', function() {
+    let selectlist;
+    const Droplist = this.Droplist;
+
+    cy
+      .getHook($hook => {
```

Comment:
Unrelated to this PR, but every time I see this I wonder when/if we should devise a Marionette/Cypress component testing setup.  In theory we could do it.. almost just like this.. but with their new component API.  Unfortunately pretty much all of it is documented for react/vue

Reply:
Ya, something worth looking into!